src/github.com/stellar/gateway/bridge/config/config.go

```diff
@@ -68,6 +68,15 @@ func (c *Config) Validate() (err error) {
 		return
 	}
 
+	for _, asset := range c.Assets {
+		if asset.Issuer == "" {
+			if asset.Code != "XLM" {
+				err = errors.New("Issuer param is required for "+asset.Code)
+				return
+			}
+		}
```

Comment:
It looks the previous version (before your PR) is missing assets validation. Could you check for other assets (if issuer is good and code is correct, [this](https://github.com/stellar/stellar-core/blob/7ac245db7c10c1d6a7fc87f7410d1e5a029c3c16/src/util/types.cpp#L61) can help for codes)?